tests/compute/test_graph.py

```diff
@@ -65,9 +65,8 @@ def _test_one(g):
             assert g.has_node(i)
             assert i in g
         assert not g.has_node(11)
-        assert not g.has_node(-1)
-        assert not -1 in g
```

Comment:
why remove these two lines?

Reply:
-1 is not being checked anymore in the new implementation because we use `int64_t` and less than operator. In fact, the negative id was not tested correctly even in the previous implementation. This is because `dgl_id_t` is `uin64_t` so -1 simply becomes very large positive integer number, making it redundant.